.clang-format

```diff
@@ -61,3 +61,4 @@ ForEachMacros:   [ foreach, Q_FOREACH, BOOST_FOREACH, forever, Q_FOREVER, QBENCH
 MaxEmptyLinesToKeep: 2
 KeepEmptyLinesAtTheStartOfBlocks: false
 
+SortIncludes: true
```

Comment:
According to the docs, this should be either `Never`, `CaseInsensitive` or `CaseSensitive`. I'd suggest to use the latter.

Reply:
Yes but it would require clang 12 or something like that...